test.ts

```
@@ -0,0 +1,14 @@
+function dayOfYear(date: string): number {
+  let [year, month, day] = date.split('-').map((item) => Number(item));
```

ESLint

[error] 'year' is never reassigned. Use 'const' instead.

(prefer-const)